### ISSUE TITLE
A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_AI4D: Controls present under ‘Tasks’ section doesn’t have Localized Control Type.

### ISSUE THREAD

**Issue by shweta255** [microsoft/azuredatastudio#9233]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#DesktopWeb;#Win10;#MAS4.1.2;#Benchmark;#A11y_SQLAzureDataStudio;#A11ySev2;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Additional Details:
MAS reference: MAS4.1.2

**Repro Steps:**
1. Launch Azure Data Studio.
2. Connect to server.
3. In the Connections pane, right-click on a server & select Manage.
4. Launch Accessibility Insights tool
5. Hover mouse over ‘New Query’ control present under ‘Tasks Section’

**Actual:**
Controls present under ‘Tasks’ section doesn’t have Localized Control Type. 

**Same issue is observed in Database Dashboard screen:** 1. Launch Azure Data Studio.2. Connect to server.3. In the Connections pane, navigate to server --> database --> right click on any database & select Manage.

**Expected:**
Controls present under ‘Tasks’ section should have Localized Control Type. 

**User Impact:**
Screen reader users will not be able to understand which control type is associated with the controls present under ‘Tasks section’.

**Attachment link reference:**
[9233_A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_AI4D.zip](https://github.com/microsoft/azuredatastudio/files/4274899/9233_A11y_SQLAzureDataStudio_ServerDashboard_ServerDashboard_AI4D.zip)

**Comment by alanrenmsft** [comment]
we are working on redesigning the dashboard, and this component will be removed. closing as won't fix

**Comment by sspatil123** [comment]
GithubTags:#Closed; Version 1.41.1
![image](https://user-images.githubusercontent.com/85863999/216965397-d4ecc4c9-974e-420a-9a5e-4ceacd4cf0e6.png)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
